src/js/schemas/job-schema/Schedule.js

```diff
@@ -23,16 +24,17 @@ const Schedule = {
       }
     },
     cron: {
-      title: "CRON Schedule",
+      title: i18nMark("CRON Schedule"),
       helpBlock: (
         <span>
-          Use cron format to set your schedule, e.g. <i>0 0 20 * *</i>
+          {i18nMark("Use cron format to set your schedule, e.g.")}{" "}
```

Comment:
Use `<Trans render="span">` instead of `i18nMark`s in this block

Reply:
What if we want to change the text?